### PR TITLE
dm: add a query-status it-test

### PR DESCRIPTION
**Pull request by docsir**
### What problem does this PR solve? 
to test GetParameters/dumpling metrics works as expected

### What is changed and how it works?
successfully worked

### Check List 

Tests 

 - Integration test

### Release note 

```release-note
`None`
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: dm/tests/many_tables/run.sh [hunk 1]
```diff
@@ -42,6 +42,13 @@ function run() {
 	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
 
 	dmctl_start_task_standalone
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
+		"\"totalTables\": \"500\"" 1 \
+		"\"completedTables\"" 1 \
```
Comment (Ehco1996) [inline dm/tests/many_tables/run.sh:48]: not sure if this load progress finished quickly, the test will never passed (completedTable >1) , so i perfer to only check `"\"totalTables\": \"500\"" 1 \`

**Review by Ehco1996 — COMMENTED** [review]
rest LGTM

File: dm/tests/many_tables/run.sh (hunk 1 above)
Comment (lance6716) [reply dm/tests/many_tables/run.sh:48]: this is test dump progress. And "only check "\"totalTables\": \"500\"" 1 \" doesn't help if dump unit is passed

Comment (Ehco1996) [reply dm/tests/many_tables/run.sh:48]: `totalTables` is in `DumpStatus`  🤔

https://github.com/pingcap/tiflow/blob/c2b9e858393a15b446e54fa473f5c4988b031831/dm/dm/proto/dmworker.proto#L104

Comment (lance6716) [reply dm/tests/many_tables/run.sh:48]: please check PR description before you review

Comment (Ehco1996) [reply dm/tests/many_tables/run.sh:48]: sorry... `"\"completedTables\"" 1 \` is not means there is one completed tables... it means on record from dmctl query-status result

**Review by Ehco1996 — APPROVED** [review]

**Pull request merged**